babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(api) {
+  api.cache.never();
+  return {
+    presets: ['@babel/preset-env'],
```

Comment:
might be worth deciding what browsers we'll support with this package and be more specific here?

Reply:
Maybe, I lean towards the approach that there is a source ES6+ version, and ES5 versions in CJS, ESM, and IIFE format.
If people want to target a specific browser set, they are welcome to import the source version and let their bundler/transpiler target their specific use case.

Reply:
See this blog post from the Babel team, for the inspiration to the current approach.
https://babeljs.io/blog/2018/06/26/on-consuming-and-publishing-es2015+-packages